src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -288,7 +288,7 @@ public function askHidden(string $question, callable $validator = null)
     /**
      * {@inheritdoc}
      */
-    public function confirm($question, $default = true)
+    public function confirm(string $question, $default = true)
```

Comment:
```suggestion
    public function confirm(string $question, bool $default = true)
```
 Is this possible too?

Reply:
I think so. The constructor of `ConfirmationQuestion` has the type.